air-quality-ui/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM node:22-alpine AS development
+
+WORKDIR /app
+
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+RUN npm ci
+
+COPY . .
+
+CMD ["npm", "run", "dev"]
+
+FROM development AS build
+
+ARG VITE_AIR_QUALITY_API_URL
+RUN npm run build
+
+FROM nginx:alpine
+
+COPY --from=build /app/.nginx/nginx.conf /etc/nginx/conf.d/default.conf
+
+WORKDIR /usr/share/nginx/html
+
+# Remove default nginx static assets
+RUN rm -rf ./*
```

Comment:
What a terrifying line. How many items does this remove?

Worth changing this to RUN rm -rf /usr/share/nginx/html/* to be doubly sure?

Reply:
I think both versions would do the same thing, it's also within the docker image so there won't be any damage to the server